examples/hello-grudge.py

```diff
@@ -0,0 +1,120 @@
+import numpy as np
+import pyopencl as cl
+from grudge.discretization import DiscretizationCollection
+import grudge.op as op
+from meshmode.mesh.generation import generate_box_mesh
+from meshmode.array_context import PyOpenCLArrayContext
+from arraycontext import freeze, thaw
+from grudge.dof_desc import DTAG_BOUNDARY
+from meshmode.discretization.connection import FACE_RESTR_INTERIOR
+
+
+ctx = cl.create_some_context()
+queue = cl.CommandQueue(ctx)
+actx = PyOpenCLArrayContext(queue)
+
+
+nel = 10
+coords = np.linspace(0, 2*np.pi, nel)
+mesh = generate_box_mesh((coords,),
+                         boundary_tag_to_face={"left": ["-x"],
+                                               "right": ["+x"]})
```

Comment:
Why does an advection equation need a "right" boundary? It seems to just be upwinding.

Reply:
For the right boundary we calculate flux similar to the interior faces. Need to include here as the interior_faces discretization (righfully) doesn't include the rightmost face of the domain.

Reply:
Ah, right, you still have an outgoing flux that needs to be handled.

Makes sense!